minio/minioadmin.py

```diff
@@ -712,3 +717,121 @@ def bucket_quota_get(self, bucket: str) -> str:
             query_params={"bucket": bucket}
         )
         return response.data.decode()
+
+    def service_account_info(self, access_key):
+        """Get information about service account"""
+        response = self._url_open(
+            "GET",
+            _COMMAND.SERVICE_ACCOUNT_INFO,
+            query_params={"accessKey": access_key},
+            preload_content=False,
+        )
+        plain_data = decrypt(
+            response, self._provider.retrieve().secret_key,
+        )
+        return plain_data.decode()
+
+    def service_account_list(self, user: str):
+        """List service accounts of user"""
+        response = self._url_open(
+            "GET",
+            _COMMAND.SERVICE_ACCOUNT_LIST,
+            query_params={"user": user},
+            preload_content=False,
+        )
+        plain_data = decrypt(
+            response, self._provider.retrieve().secret_key,
+        )
+        return plain_data.decode()
+
+    def service_account_add(self,
+                            access_key: str = "",
+                            secret_key: str = "",
```

Comment:
1. access/secret key args should be `access_key: str | None = None` and `secret_key: str | None = None`
2. Add typing to return value